pulse2percept/implants/electrode_arrays.py

```diff
@@ -179,7 +179,7 @@ def plot(self, annotate=False, autoscale=True, ax=None, color_stim=None, cmap='O
                               'fc': (1, 1, 1, 0.7)},
                         zorder=ZORDER['annotate'])
         patch_collection = PatchCollection(patches, match_original=True,
-                                          zorder=ZORDER['annotate'], cmap=cm, norm=norm)
+                                          zorder=ZORDER['foreground'], cmap=cm, norm=norm)
         ax.add_collection(patch_collection)
```

Comment:
I don't get why this stopped working. "annotate" has zorder=2000, and "foreground" has zorder=50. There's also "front" that could be used with zorder=9999

Reply:
I think the problem was that both the patches and the annotations were registered with the annotate zorder before, so the patches went on top because that happens later. 

I changed it so that the patches are in the foreground now instead of annotate layer

Reply:
Oh, gotcha. That makes sense then